docs/adr/index.md

```diff
@@ -1,14 +0,0 @@
-# Architectural Decision Log
```

Comment:
Did a pre-commit hook delete this? We probably want this file added back again

Reply:
I mentioned this in the original [EASi PR](https://github.com/CMSgov/easi-app/pull/1727#issuecomment-1209610374), but I can echo the main points here:

- The git hook to actually update this automatically (turns out it was supposed to be automatic) was broken.
- I am not sure the index was actually that useful, since it kinda just echoed the filesystem, and the files are named clearly enough as-is

So I ended up just deleting it on purpose 😂

Reply:
thanks Clay! Makes sense, I think what you did for EASI, linking directly the folder directory instead of the index will work great :).